llvm/utils/TableGen/SubtargetEmitter.cpp

```diff
@@ -1610,9 +1610,9 @@ static void emitPredicates(const CodeGenSchedTransition &T,
     }
 
     SS << ")\n"; // end of if-stmt
-    PE.decreaseIndentLevel();
-    SS.indent(PE.getIndentLevel() * 2);
-    PE.decreaseIndentLevel();
+    PE.getIndent()--;
```

Comment:
Use prefix form? Or does the function call interfere with that?

Reply:
Done. Yeah, no reason to use postfix form.